src/Components/Footer/index.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
```

Comment:
Folder name should be camelCase i.e fooBar.

Only component fileName can be uppercase i.e FooBar.js

Also, create Footer.js and export from index.js.